superset/charts/scribe_post_processing.py

```diff
@@ -34,6 +34,8 @@ def process_col(col):
 
 def apply_scribe_post_process(data, is_csv_format):
     if is_csv_format:
+        if data == "\n":
```

Comment:
We don't need to worry about completely empty data, or anything else except "\n"?

Reply:
From what I've played with it we always get here "\n" for empty table, and then parsing fails because of empty header. 
I've tried digging into how this is created by superset but it seems not worth the time